Changes to be committed:

terizhao opened the pull request:
modified:   docs/locale/zh_CN/source/discovery-overview.rst

terizhao committed:
Changes to be committed:
	modified:   docs/locale/zh_CN/source/discovery-overview.rst

zRich commented on docs/locale/zh_CN/source/discovery-overview.rst:
```diff
@@ -4,67 +4,82 @@
 ---------------------------------
 
-为了在 Peer 节点上执行链码，将交易提交给排序节点，并更新交易状态，应用程序需要连接 SDK 中开放的 API。
+为了在对等节点上执行链码、向订购节点提交交易并获取交易状态的更新，应用程序通过一个由SDK公开的API进行连接。
```
orderer在fabric里面是排序的意思。

terizhao replied:
address it in the new commit.

zRich commented on docs/locale/zh_CN/source/discovery-overview.rst:
```diff
@@ -4,67 +4,82 @@
+为了在对等节点上执行链码、向订购节点提交交易并获取交易状态的更新，应用程序通过一个由SDK公开的API进行连接。
 
-然而，为了让应用程序连接到相关的网络节点，SDK 需要大量信息。除了通道上排序节点和 Peer 节点的 CA 证书和 TLS 证书及其 IP 地址和端口号之外，它还必须知道相关的背书策略以及安装了链码的 Peer 节点（这样应用程序才能知道将链码提案发送给哪个 Peer 节点）。
+然而，SDK需要大量的信息，以便让应用程序能够连接到相关的网络节点。除了通道上订购者和对等节点的CA和TLS证书,
```
orderer service翻译成排序服务，orderer peer/node 翻译成排序节点

terizhao replied:
address it in the new commit

terizhao committed:
Changes to be committed:
	modified:   docs/locale/zh_CN/source/CONTRIBUTING.rst

terizhao committed:
Changes to be committed:
	modified:   docs/locale/zh_CN/source/discovery-overview.rst
        addressed Rich's comments
